specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2021-06-08/DataLakeStorage.json

```diff
@@ -3165,6 +3168,15 @@
         "name": "cpk-info"
       },
       "description": "The algorithm used to produce the encryption key hash. Currently, the only accepted value is \"AES256\". Must be provided if the x-ms-encryption-key header is provided."
+    },
+    "Flush": {
+      "name": "flush",
+      "x-ms-client-name": "flush",
```

Comment:
I don't think `x-ms-client-name` is needed since it doesn't differ from the `name`. You can also define the parameter inline (since it is only being used in one place) to avoid having to specify the `x-ms-parameter-location`. We should specify the `default` value (which presumably is `false`?)

Reply:
I included the `x-ms-client-name` to follow the convention from the rest of the file, it appears every parameter has `x-ms-client-name` defined.  I also didn't define the parameter in-line to pattern match the rest of the file.  It appears no other boolean parameter in the this file defines a default.